docs_src/microservices/application/sdk/GettingStarted.md

```diff
@@ -97,7 +95,7 @@ if err := service.SetDefaultFunctionsPipeline(
     ...
 }
 ```
-Set the Trigger type to `http` in configuration file found here: [res/configuration.yaml](https://github.com/edgexfoundry/edgex-examples/blob/{{edgexversion}}/application-services/custom/simple-filter-xml/res)
+Set the Trigger type to `http` in configuration file found here: [res/configuration.yaml](https://github.com/edgexfoundry/edgex-examples/blob/{{edgexversion}}/application-services/custom/simple-filter-xml/res/configuration.yaml)
 
 ```toml
```

Comment:
should this be converted to yaml?

Reply:
Already converted. It just wasn't linked all the way to the yaml file.

Reply:
I was actually looking at the toml below the linked file.

Reply:
ahhh, in-line ... got it.

Reply:
fixed